src/main/java/io/cdap/e2e/pages/locators/CdfControlCenterLocators.java

```diff
@@ -0,0 +1,167 @@
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Back')]")
+  public static WebElement backTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//*[@class='icon-svg icon-close']")
+  public static WebElement closeTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying all entities, sorted by Newest')]")
+  public static WebElement datasetApplicationsArtifactsMessage;
+
+  public static By locateDropdownListItem(String option) {
+    return By.xpath("//input[@data-cy='" + option + "']");
+  }
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Schema')]")
+  public static WebElement schemaTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//a[contains(text(),'Programs')]")
+  public static WebElement programsTabDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='schema-tab']")
+  public static WebElement schemaTabDatasetEntityPage;
+
+  @FindBy(how = How.XPATH, using = "//div[@class='program-tab clearfix']")
+  public static WebElement programsTabDatasetEntityPage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by Newest')]")
+  public static WebElement newestOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by Oldest')]")
+  public static WebElement oldestOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by Z - A')]")
+  public static WebElement zToAOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Displaying Applications, Datasets, sorted by A - Z')]")
+  public static WebElement aToZOptionMessage;
+
+  @FindBy(how = How.XPATH, using = "//div[@id='filter-tooltip-target-id']")
+  public static WebElement sortDropdown;
+
+  public static WebElement selectSortDropdownValue(String option) {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//*[contains(text(),'" + option + "')]"));
+  }
+
+  @FindBy(how = How.XPATH, using = "//span[@data-testid='tag-plus-button']")
+  public static WebElement addTagIconDatasetEntity;
+
+  public static WebElement enterAddTagIconDatasetEntityValue() {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//input[@data-testid='tag-input']"));
+  }
+
+  @FindBy(how = How.XPATH, using = "//*[contains(text(),'Tags (1)')]")
+  public static WebElement tagCountIconIncreaseDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//span[@class='tag-content']//*[@class='icon-svg icon-close']")
+  public static WebElement closeTagIconDatasetEntity;
+
+  @FindBy(how = How.XPATH, using = "//i[normalize-space()='No tags found. Click to add a new business tag.']")
+  public static WebElement tagCounDecreaseIconDatasetEntityMessage;
+
+  @FindBy(how = How.XPATH, using = "//span[contains(text(),'Search results for \"testingtag\", filtered by Appli')]")
+  public static WebElement searchedTagShowsDisplayedMessage;
```

Comment:
do not locate elements using text content.

Reply:
Changed the above both.